File: serverApp/src/main/java/com/server/dto/EditGuideDTO.java
package com.server.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class EditGuideDTO {
    private Long id;

    private String title;

    private String contents;
}

File: guide_app/lib/common/dto/edit_guide_dto.dart
import 'package:json_annotation/json_annotation.dart';

part 'edit_guide_dto.g.dart';

/// Edit guide dto.
/// For code generation run  flutter pub run build_runner build --delete-conflicting-outputs.
@JsonSerializable()
class EditGuideDto {
  EditGuideDto(this.id, this.title, this.content);
  final int id;
  final String title;
  @JsonKey(name: 'contents')
  final String content;

  Map<String, dynamic> toJson() => _$EditGuideDtoToJson(this);
}